core/lib/dr_tools.h

```diff
@@ -48,6 +48,12 @@ DR_API
  * \warning This context cannot be used as the drcontext for a thread
  * running under DR control!  It is only for standalone programs that
  * wish to use DR as a library of disassembly, etc. routines.
+ * \warning This context is not fully thread-safe as it stores
+ * the current ISA mode DR uses in a global variable.
```

Comment:
Use an explicit link so it's clear what we're talking about: `dr_set_isa_mode()`, not `instr_set_isa_mode()`.

Reply:
Done.